### ISSUE TITLE
SmbServerNameHardeningLevel policy does not apply

### ISSUE THREAD

**Issue by kewalaka** [PowerShell/SecurityPolicyDsc#85]
Due to an empty space at the end of the string on this line:

https://github.com/PowerShell/SecurityPolicyDsc/blob/68ffb4a9e9f5e7d2834f4d0cedad0e878bca938d/DSCResources/MSFT_SecurityOption/SecurityOptionData.psd1#L412

throws:

```
VERBOSE: [VAGRANT]: LCM:  [ End    Set      ]  [[SecurityOption]NetworkSecurityOptions]  in 1.8370 seconds.
PowerShell DSC resource MSFT_SecurityOption  failed to execute Set-TargetResource functionality with error message: Failed to update security option 
Microsoft_network_server_Server_SPN_target_name_validation_level. Refer to %windir%\security\logs\scesrv.log for details. 
    + CategoryInfo          : InvalidOperation: (:) [], CimException
    + FullyQualifiedErrorId : ProviderOperationExecutionFailure
    + PSComputerName        : localhost
 
VERBOSE: [VAGRANT]: LCM:  [ End    Set      ]
The SendConfigurationApply function did not succeed.
    + CategoryInfo          : NotSpecified: (root/Microsoft/...gurationManager:String) [], CimException
    + FullyQualifiedErrorId : MI RESULT 1
    + PSComputerName        : localhost
```

**Comment by jcwalker** [comment]
@kewalaka this is fixed in the dev branch.  When it gets merged into master I will close the issue.  Please test to confirm the fix. Thanks and nice catch!

**Comment by johlju** [comment]
@heatherdaye Could you please open a separate issue for that? Thanks

**Comment by jcwalker** [comment]
Closes this as this issue was resolved in version 2.5.0.0.
